MASH-FRET/source/traces/export/openExpTtpr.m

```diff
@@ -1102,7 +1176,12 @@ function popup_molTagged_Callback(obj, evd, h_fig)
 
 
 function pushbutton_infos_Callback(obj, evd, h_fig)
```

Comment:
I really like this functionality! Ideally we should have a small info icon in every panel of the GUI to link to the documentation (e.g. an icon like this: https://fontawesome.com/icons/info-circle?style=solid). This would help the user in navigating the docs and find the appropriate help page more easily.

Reply:
Totally agreed, I'll do this if I have time left, otherwise I leave it up to you guys

Reply:
Oh well it's in now, I couldn't leave it undone. You may change the help icon, I just dray a question mark